### PR TITLE
Update README.md

### LINKED ISSUE #99: Entity Prefabs
## Reason
To support an ergonomic method for adding multiple components at once to a new or already existing entity.

## Requirements
- [ ] It should be possible for a user of the framework to create a custom prefab that allow them to add an entity with a predetermined set components.
- [ ] It should be possible to use a prefab multiple times, to create multiple entities.
- [ ] A user should be able to create a "bundle" prefab that contain a set of components which can be added to an already existing entity all at once.

## Dependencies
- [ ] \#issue number

### PR DESCRIPTION
**Pull request by martinjonsson01**
# blalbal
good issue

Closes #99

### CODE REVIEW

**Review by linnosterlund — CHANGES_REQUESTED** [review]
good

**Review by ChristofferPerssons — APPROVED** [review]
bad

File: README.md [hunk 1]
```diff
@@ -10,3 +10,6 @@ This project uses Rust nightly.
 2. Run `pip install pre-commit`
 3. In the root directory of the project, run `pre-commit install`
 4. Run `pre-commit run --all-files` to verify that it's working
+
+
+Hello
```
Comment (JacobBredin) [inline README.md:15]: remov

**Review by JacobBredin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JacobBredin) [inline README.md:15]:
```suggestion


Hello
```

Comment (linnosterlund) [inline README.md:15]: Please dont

```suggestion



```

**Review by linnosterlund — CHANGES_REQUESTED** [review]

**Pull request closed without merging**